go/teams/loader.go

```diff
@@ -308,6 +308,11 @@ func (l *TeamLoader) load1(ctx context.Context, me keybase1.UserVersion, lArg ke
 		}
 	}
 
+	newMctx, shouldReload := VerifyBoxAudit(libkb.NewMetaContext(ctx, l.G()), teamID)
```

Comment:
@mlsteele this is going to happen every team load so wanted to get your opinion on this (this is in FTL too). In the normal case it hits a LRU cache that says the team is not in jail and returns normally. On startup it needs to get the value from disk, and then fill the LRU cache with that info, so subsequent calls will be faster.

When the team is in jail (failed 6 audits spread out over 42 hours), however, which might happen if you're offline for several days in a row, or you were deleted from a team and readded later, or the server is malicious, it first tries to load the team, and if the team is not open and user is a writer or greater, rotate the team and then do a reaudit. If the reaudit fails, a blackbar appears. shouldReload is true because the team has now changed, so the load happens again, but with a context that will no-op the VerifyBoxAudit. If reaudit fails the team load still works, you just get a blackbar everytime you try to load the team (until it passes reaudit).

Totally open to making changes or changing how all of this works, let me know.

Reply:
Ok. Using the context value to deal with reentrancy seems pretty nice compared to passing a parameter all the way down.

Seems like a failed audit should be able to cause the team to fail to load. I'm not seeing how that would happen here.

Reply:
It doesn't right now because this code might be buggy and not play well with users who don't have puks or who have reset (though I have tested both cases, might be others). It manifests as constant blackbars in the app whenever you do anything with the team.
It might even be a good idea to put this behind a server driven feature flag for a few weeks. But eventually we could make it so it just returns an error instead.

Reply:
Guess I was misunderstanding. I took black bars to mean team loads failed which caused operations to fail which caused black bars. But if you mean a special message is routed to the ui dedicated to showing black bars, but also everything just works.. Then ok makes sense for starters.

But watch out for the issue where when the ui gets a barrage of black bars the black bar just turns black and stops rendering text.